<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_encode">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>编码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_src">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>源文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>编码文件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_encodeFileName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>encode.dat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>解码文件：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_decodeFileName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>decode.dat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decode">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>解码</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_width">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>width</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_width">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_height">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_height">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>255</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_bitrate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>bitrate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_bitrate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>40960000</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_fps">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fps">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>25</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_gopSize">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gop_size</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_maxBFrame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>max_bframe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_srcFmt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>srcFmt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_destFmt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>destFmt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_codecID">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>codecID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_gopsize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_maxbframe">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_srcFmt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GRAY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H264</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_destFmt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GRAY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>YUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H264</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_codecID">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AV_CODEC_ID_H264</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_width</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_width</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_height</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_height</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_maxBFrame</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_maxbframe</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_gopSize</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_gopsize</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_fps</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_fps</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_bitrate</sender>
   <signal>clicked(bool)</signal>
   <receiver>lineEdit_bitrate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_srcFmt</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_srcFmt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_destFmt</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_destFmt</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_codecID</sender>
   <signal>clicked(bool)</signal>
   <receiver>comboBox_codecID</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
